Snabble/UI/Scanner/RecommendedZoom.swift

```diff
@@ -25,7 +25,6 @@ public enum RecommendedZoom {
     static let minimumCodeSize: Float = 40 // in mm, this is the recommended size of an EAN-13 barcode
 
     public static func factor(for videoInput: AVCaptureDeviceInput) -> Float {
-        #if compiler(>=5.5) // this only works in Xcode 13
```

Comment:
@gereons sollten wir es nicht drin behalten, damit es weiterhin Xcode 12 kompatibel ist?

Reply:
Ich glaube nicht dass wir diese Kompatibilität brauchen - allerdings würde ich auch den Branch erst mergen wenn wir wissen, ob wir den Workaround aus https://github.com/sindresorhus/Defaults/issues/81 auch bei uns machen müssen, oder das Issue vielleicht in der Appstore-Releaseversion tatsächlich gefixt ist.